identity-credential/src/presentation/presentation.rs

```diff
@@ -49,6 +52,9 @@ pub struct Presentation<T = Object, U = Object> {
   /// Miscellaneous properties.
   #[serde(flatten)]
   pub properties: T,
+  /// Proof(s) used to verify a `Presentation`
+  #[serde(skip_serializing_if = "Option::is_none")]
+  pub proof: Option<Signature>,
```

Comment:
See comment above regarding `flatten`. I think ordering is important here (for the generated code)